src/analysis/convex/cone.lean

```diff
@@ -351,6 +353,10 @@ def positive_cone : convex_cone 𝕜 E :=
     end,
   add_mem' := λ x (hx : _ ≤ _) y (hy : _ ≤ _), add_nonneg hx hy }
 
+@[simp] lemma mem_positive_cone {x : E} : x ∈ positive_cone 𝕜 E ↔ 0 ≤ x := iff.rfl
+
+lemma positive_cone_eq_Ici : (positive_cone 𝕜 E : set E) = Ici 0 := rfl
```

Comment:
This exists already in #15837

Reply:
Ah I see, thanks! I'll use that one once it merges.

Reply:
Updated.